static/app/views/issueDetails/metricIssues/useMetricSessionStats.tsx

```diff
@@ -38,7 +38,7 @@ export function useMetricSessionStats(
     {
       query: {
         project: project.id ? [Number(project.id)] : [],
-        environment,
+        environment: environment ? environment : undefined,
```

Comment:
What does `undefined` make the query param look like?

Reply:
undefined removes it as part of how query-string encodes query parameters